Samples/PublicSamples/EchoBot/src/EchoBot/Program.cs

```diff
@@ -1,64 +1,21 @@
-﻿using EchoBot.Services.ServiceSetup;
-using Microsoft.Extensions.Configuration;
-using System;
-using System.Diagnostics;
-using System.Reflection;
-using System.Threading.Tasks;
-using Microsoft.ApplicationInsights.Channel;
+using EchoBot;
+using Microsoft.Extensions.Logging.Configuration;
+using Microsoft.Extensions.Logging.EventLog;
 
-namespace EchoBot
-{
-    class Program
+IHost host = Host.CreateDefaultBuilder(args)
+    .UseWindowsService(options =>
     {
-        /// <summary>
-        /// Defines the entry point of the application.
-        /// </summary>
-        /// <param name="args">The arguments.</param>
-        public static async Task Main(string[] args)
```

Comment:
Could you explain the change here, why we are getting rid of the earlier entry point for the service, class and namespace?

Reply:
Now that the libraries support dotnet 6 we can use a background service which runs as a windows service. You can also run this locally and there is no need to have separate projects. Also, moved away from using Owin and now using WebApplication builder to create the web app.